lang/lang_en_fr.txt

```diff
@@ -348,7 +348,7 @@
 
 #MSG_RESPONSE_POOR c=20 r=2
 "Fil. sensor response is poor, disable it?"
-"La reponse du capteur de fil. est pauvre, le desactiver ?"
+"La reponse du capteur de filament est pauvre, le desactiver ?"
```

Comment:
This is too long as shown above c=20 r=2 means max. 20 characters per row and max 2 rows. This translation will use 3 rows